@types/webgl2 dependency conflicts with TS built-in types for WebGL

### Description

I tried adding @google/model-viewer to my typescript project. Because there's a dependency on @types/webgl2, it causes this build error. A pretty good [summary of the issue is here](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/40995).

```
node_modules/typescript/lib/lib.dom.d.ts:16532:11 - error TS2430: Interface 'WebGL2RenderingContext' incorrectly extends interface 'WebGLRenderingContext'.
  Types of property 'clearBufferfv' are incompatible.
    Type '(buffer: number, drawbuffer: number, values: ArrayLike<number> | Float32Array, srcOffset?: number) => void' is not assignable to type '(buffer: number, drawbuffer: number, values: ArrayBufferView, srcOffset: number) => void'.
      Types of parameters 'values' and 'values' are incompatible.
        Type 'ArrayBufferView' is not assignable to type 'ArrayLike<number> | Float32Array'.
          Type 'ArrayBufferView' is missing the following properties from type 'Float32Array': BYTES_PER_ELEMENT, copyWithin, every, fill, and 22 more.

16532 interface WebGL2RenderingContext extends WebGL2RenderingContextBase, WebGL2RenderingContextOverloads, WebGLRenderingContextBase {
                ~~~~~~~~~~~~~~~~~~~~~~


Found 1 error.
```

--- a/packages/3dom/src/api.ts
+++ b/packages/3dom/src/api.ts
@@ -193,9 +193,12 @@ export declare type Constructor<T = object> = {
   new (...args: any[]): T; prototype: T;
 };
 
+/**
+ * A constructor that accepts a specific set of arguments during construction
+ */
 export declare type ConstructedWithArguments<T extends unknown[] = unknown[]> =
     {
-      new (...args: T): unknown;
+      new (...args: T): object;
     };
 
 /**

--- a/packages/model-viewer/package.json
+++ b/packages/model-viewer/package.json
@@ -110,8 +110,7 @@
     "@google/3dom": "0.0.1-alpha.3",
     "lit-element": "^2.2.0",
     "three": "^0.113.2",
-    "@types/resize-observer-browser": "^0.1.2",
-    "@types/webgl2": "^0.0.5"
+    "@types/resize-observer-browser": "^0.1.2"
   },
   "publishConfig": {
     "access": "public"

